com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1077,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- GWT Plugin debug launching -->
+   <extension
+         point="com.google.gdt.eclipse.core.debugLauncher">
+      <extensions
+            class="com.github.sdbg.debug.DebugLaunch">
+      </extensions>
+   </extension>
+   
```

Comment:
See previous comment. Let's move this extension point contribution to the plugin.xml of plugin com.github.sdbg.integration.jdt